ci/install.sh

```diff
@@ -14,12 +14,10 @@ fi
 
 # Use default CC to avoid compilation problems when installing Python modules.
 echo "Install neovim module and coveralls for Python 2."
-CC=cc pip2.7 -q install --user --upgrade neovim cpp-coveralls
+CC=cc pip2 -q install --user --upgrade neovim cpp-coveralls
 
 echo "Install neovim module for Python 3."
-# Allow failure. pyenv pip3 on travis is broken:
-# https://github.com/travis-ci/travis-ci/issues/8363
```

Comment:
That issue is still open. What changed?

Reply:
I'm still playing around with a few solutions. But I'll add the URL to that issue above the `pyenv globals` line.

Reply:
Fwiw It does work sometimes, check multiple builds. And the git log history for these lines.